cmd/code/scan.go

```diff
@@ -31,6 +33,8 @@ func newScanCommand() *cobra.Command {
 	cmd.Flags().StringVar(&dbPath, "db", "", "Path to create the sqlite database")
 	cmd.Flags().StringArrayVar(&appDirs, "app", []string{"."}, "Directories to scan for application code files")
 	cmd.Flags().StringArrayVar(&importDirs, "import-dir", []string{}, "Directories to scan for import files")
+	cmd.Flags().StringArrayVarP(&excludePatterns, "exclude", "", []string{},
```

Comment:
Whats the rationale behind this? Do we also include a `include` pattern then? If so we need to decide on the precedence

Reply:
its just to ignore irrelevant things, like we have lot of code examples with vulnerabilities in say, `test/fixtures`
when building a code database, i would just do `vet code scan --app . --db vet.db --exclude test/fixtures`
so file system never enumerates these fixtures which otherwise would create false positive deps usage